### PR TITLE
Remove noise from the template list

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

File: docs/sdk/src/polkadot_sdk/templates.rs [hunk 1]
```diff
@@ -7,34 +7,23 @@
...
+//! - [`parachain-template`](https://github.com/paritytech/polkadot-sdk-solochain-template): 
+//!   Formerly known as "substrate-node-template", is a white-labeled substrate-based blockchain 
+//!   (aka. solochain) that contains moderate features, such as a basic consensus engine and some 
+//!   FRAME pallets. This template can act as a good starting point for those who want to launch 
+//!   a solochain.
+//! - [`parachain-template`](https://github.com/paritytech/polkadot-sdk-solochain-template): 
+// A parachain template ready to be connected to a relay-chain, such as [Paseo]
+//! (https://github.com/paseo-network/.github), Kusama  or Polkadot.
+//!
+//! Note that these templates are mirrored automaticall from [this]
```
Comment (pepoviola) [inline docs/sdk/src/polkadot_sdk/templates.rs:24]:
```suggestion
//! Note that these templates are mirrored automatically from [this]
```

**Review by pepoviola — APPROVED** [review]

File: docs/sdk/src/polkadot_sdk/templates.rs [hunk 2]
```diff
@@ -7,34 +7,23 @@
...
-//! - [`extended-parachain-template`](https://github.com/paritytech/extended-parachain-template): A
-//!   parachain template that contains more built-in functionality such as assets and NFTs.
-//! - [`frontier-parachain-template`](https://github.com/paritytech/frontier-parachain-template): A
-//!   parachain template for launching EVM-compatible parachains.
+//! The following templates are maintained as a part of the `polkadot-sdk` repository:
+//!
+//! - [`minimal-template`](https://github.com/paritytech/polkadot-sdk-minimal-template): 
+//!   A minimal template that contains the least amount of features to be a functioning blockchain. 
+//!   Suitable for learning and testing.
+//! - [`parachain-template`](https://github.com/paritytech/polkadot-sdk-solochain-template):
```
Comment (muharem) [inline docs/sdk/src/polkadot_sdk/templates.rs:15]: should the link be named as - `solochain-template` ?

File: docs/sdk/src/polkadot_sdk/templates.rs [hunk 3]
```diff
@@ -7,34 +7,23 @@
...
+//!
+//! - [`minimal-template`](https://github.com/paritytech/polkadot-sdk-minimal-template): 
+//!   A minimal template that contains the least amount of features to be a functioning blockchain. 
+//!   Suitable for learning and testing.
+//! - [`parachain-template`](https://github.com/paritytech/polkadot-sdk-solochain-template): 
+//!   Formerly known as "substrate-node-template", is a white-labeled substrate-based blockchain 
+//!   (aka. solochain) that contains moderate features, such as a basic consensus engine and some 
+//!   FRAME pallets. This template can act as a good starting point for those who want to launch 
+//!   a solochain.
+//! - [`parachain-template`](https://github.com/paritytech/polkadot-sdk-solochain-template):
```
[the remark this answers is not in the document]
Comment (kianenigma) [reply docs/sdk/src/polkadot_sdk/templates.rs:20]: damn, thanks for catching these!

**Review by kianenigma — COMMENTED** [review]
Fix mistakes

**Review by muharem — APPROVED** [review]

File: docs/sdk/src/polkadot_sdk/templates.rs [hunk 4]
```diff
@@ -7,34 +7,22 @@
...
+//! - [`minimal-template`](https://github.com/paritytech/polkadot-sdk-minimal-template): 
+//!   A minimal template that contains the least amount of features to be a functioning blockchain. 
+//!   Suitable for learning and testing.
+//! - [`solochain-template`](https://github.com/paritytech/polkadot-sdk-solochain-template): 
+//!   Formerly known as "substrate-node-template", is a white-labeled substrate-based blockchain 
+//!   (aka. solochain) that contains moderate features, such as a basic consensus engine and some 
+//!   FRAME pallets. This template can act as a good starting point for those who want to launch 
+//!   a solochain.
+//! - [`parachain-template`](https://github.com/paritytech/polkadot-sdk-parachain-template): 
+// A parachain template ready to be connected to a relay-chain, such as [Paseo]
```
Comment (lexnv) [inline docs/sdk/src/polkadot_sdk/templates.rs:21]: nit: Is the question mark missing here intended?

```suggestion
//! A parachain template ready to be connected to a relay-chain, such as [Paseo]
```

**Review by lexnv — APPROVED** [review]
LGTM! Nice 🙏

**Comment by kianenigma** [comment]
bot fmt

**Pull request merged**